Xamarin.PropertyEditing/ViewModels/PropertyViewModel.cs

```diff
@@ -23,7 +23,7 @@ public PropertyViewModel (IPropertyInfo property, IEnumerable<IObjectEditor> edi
 			UpdateCurrentValue ();
 		}
 
-		public ValueSource ValueSource => this.value.Source;
```

Comment:
`ValueSource` does not combine, this should just be `ValueSource.Default`. Also let's add a test that this property is `Default` when there's no value source (like when two editors are present and giving different values.)